SUMMARY.md

```diff
@@ -102,6 +102,7 @@
         * [Add cart validator](guides/plugins/plugins/checkout/cart/add-cart-validator.md)
         * [Change price of items in cart](guides/plugins/plugins/checkout/cart/change-price-of-item.md)
         * [Customize price calculation](guides/plugins/plugins/checkout/cart/customize-price-calculation.md)
+        * [Use nested line items](guides/plugins/plugins/checkout/cart/use-nested-line-items.md)
```

Comment:
Shouldn't this be a storefront guide?
If so, can you please move the file and change the entry in the SUMMARY.md?